charts/karmada/values.yaml

```diff
@@ -11,6 +11,11 @@ global:
   ##   - myRegistryKeySecretName
   imagePullSecrets: []
 
+## @karmadaImageVersion the image version of each karmada components
+## "latest" is a default value for local installation
+## specific version for remote installation, which is defined by hack/package-helm-chart.sh when releasing
+karmadaImageVersion: &karmadaImageVersion latest
```

Comment:
I'm not familiar with this usage, can you provide a link to indicate if it is appropriate?

Reply:
This is the syntax for yaml, you can search with key words `anchors and aliases of YAML`

Or you can refer to: [anchors-and-aliases](https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_advanced_syntax.html#yaml-anchors-and-aliases-sharing-variable-values)

Reply:
```
## @karmadaImageVersion the image version of each karmada components
## "latest" is a default value for local installation.
## If update this anchors("&karmadaImageVersion"), also update hack/package-helm-chart.sh.
## See https://yaml.org/spec/1.2/spec.html#id2765878 and
## https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_advanced_syntax.html#yaml-anchors-and-aliases-sharing-variable-values
## for more details about yaml anchors and aliases.
```
Thanks for the update, how about putting the link to the comments?

Reply:
ok, done